sdk/eventhub/event-hubs/test/eventProcessor.spec.ts

```diff
@@ -96,4 +100,58 @@ describe("Event Processor", function(): void {
       isCloseCalled.should.equal(true);
     });
   });
+
+  describe("InMemory Partition Manager", function(): void {
+    const inMemoryPartitionManager = new InMemoryPartitionManager();
+
+    it("should claim ownership ", async function(): Promise<void> {
+      const partitionOwnership1: PartitionOwnership = {
+        eventHubName: "myEventHub",
+        consumerGroupName: EventHubClient.defaultConsumerGroupName,
+        instanceId: generate_uuid(),
+        partitionId: "0",
+        ownerLevel: 10
+      };
+      const partitionOwnership2: PartitionOwnership = {
+        eventHubName: "myEventHub",
+        consumerGroupName: EventHubClient.defaultConsumerGroupName,
+        instanceId: generate_uuid(),
+        partitionId: "1",
+        ownerLevel: 10
+      };
+      const partitionOwnership = await inMemoryPartitionManager.claimOwnerships([
+        partitionOwnership1,
+        partitionOwnership2
+      ]);
+      partitionOwnership.length.should.equals(2);
+    });
+
+    it("should get a list of ownerships ", async function(): Promise<void> {
+      const partitionOwnershipList = await inMemoryPartitionManager.listOwnerships(
+        "myEventHub",
+        EventHubClient.defaultConsumerGroupName
+      );
+      partitionOwnershipList.length.should.equals(2);
```

Comment:
This is 2 because of what was done in the previous test?
Which means that if this test was run independently, then it would fail?
If that is true, then we should combine all checks into a single test case, because each test should be runnable on its own